resources/providers/PingIdentity/Provider.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace SocialiteProviders\PingIdentity;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'PINGIDENTITY';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'openid',
+        'profile',
+        'email'
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getAuthUrl($state)
+    {
+	$auth_url = $this->buildAuthUrlFromBase($this->getInstanceUri().'/as/authorization.oauth2', $state);
+	$auth_url .= "&acr_values=Single_Factor&prompt=login";
+	return $auth_url;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getTokenUrl()
+    {
+	return $this->getInstanceUri() . '/as/token.oauth2?acr_values=Single_Factor&prompt=login';
+    }
```

Comment:
It looks like a bunch of places in this file are indented with tabs instead of spaces.  Most of the codebase uses 4-space indentation, so it would be good to use spaces here too.  It seems like we don't have a rule to enforce it yet, but we should probably add one in the future.